src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,23 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
```

Comment:
Ensure that the `Injector` class is correctly implemented and configured to provide instances of the services. If the `Injector` class is not properly set up, it could lead to runtime errors when trying to get instances of the services.